tests/regression/runner.sh

```diff
@@ -0,0 +1,28 @@
+# autorun regression test
+
+if [ -z $1 ]; then
+    ta_host="localhost"
+else
+    ta_host=$1
+fi
+
+if [ -z $2 ]; then
+    ta_port="5566"
+else
+    ta_port=$2
+fi
+
+if [ -z $3 ]; then
+    iri_host="localhost"
+else
+    iri_host=$3
+fi
+
+if [ -z $4 ]; then
+    iri_port="14265"
+else
+    iri_port=$4
+fi
+
+bazel-bin/accelerator/accelerator --ta_host=$1 --ta_port=$2 --iri_host=$3 --iri_port=$4 &
```

Comment:
This isn't the way I said. Idealy every tests should run under bazel structure, that means rules for python script and also rules for shell script.

Reply:
I will use bazel to run this regression test.